pom.xml

```diff
@@ -856,6 +864,9 @@
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
               <testFailureIgnore>true</testFailureIgnore>
+            	<parallel>classes</parallel>
```

Comment:
Please fix formatting to match the line above - use 2 instead 3 spaces. Same for the line below